### PR TITLE
Rewritten kiwix-manage man

### LINKED ISSUE #574: Provide a sample library.xml file
The website mentions that there's a sample file at https://www.kiwix.org/library.xml, but the link appears to be broken.

### LINKED ISSUE #575: Man page for kiwix-manage is outdated
I was confused by the "url" option mentioned in the man page for kiwix-manage. I expected that kiwix-manage would download and add a ZIM file to its library with this option. However, this option doesn't exist in the latest kiwix-tools.

There is a `--url` flag for the "add" option, which adds a hyperlink to the original source of the ZIM file in the library file.

There are 3 flags mentioned in the man page which are not in the source code anymore: `--current`, `--backend` and `--indexPath`

### PR DESCRIPTION
**Pull request by kelson42**
Fixes #574
Fixes #575

### CODE REVIEW

File: src/man/kiwix-manage.1 [hunk 1]
```diff
@@ -1,62 +1,72 @@
...
 
 .TP
-\fB\-\-backend=xapian|clucene\fR
-Select an indexing backend.
+Other options (to be used alone):
 
 .TP
-\fB\-\-indexPath=FULLTEXT_IDX_PATH\fR
-Path to full-text index for that ZIM file.
+\fB\-\-help | \-h
```
Comment (veloman-yunkan) [inline src/man/kiwix-manage.1:58]: No closing `\fR`

File: src/man/kiwix-manage.1 [hunk 2]
```diff
@@ -1,62 +1,72 @@
...
+Display the kiwix-manage help
 
 .TP
-\fBurl\fR
-Set the content location of the ZIM file over the network for in\-kiwix download.
+\fB\-\-version | \-v
```
Comment (veloman-yunkan) [inline src/man/kiwix-manage.1:62]: No closing `\fR`

File: src/man/kiwix-manage.1 [hunk 3]
```diff
@@ -1,62 +1,72 @@
...
+.SH OPTIONS
 .TP
-\fBremove\fR
-Remove the given \fBCONTENT_ID\fR from \fBLIBRARY_FILE\fR. At least one \fBCONTENT_ID\fR should be specified.
+Options to be used with the action \fBadd\fR:
 
 .TP
-\fB\-\-zimPathToSave=\fR
-Change the location reference of the ZIM file in the library.
+\fB\-\-url=HTTP_URL\fR
```
Comment (veloman-yunkan) [inline src/man/kiwix-manage.1:47]: What about displaying `HTTP_URL` in regular or italic font?

File: src/man/kiwix-manage.1 [hunk 4]
```diff
@@ -1,62 +1,72 @@
...
+Set the ZIM online HTTP(S) URL
 
 .TP
-\fB\-\-current\fR
-Mark this content as the current (default) content in the library.
+\fB\-\-zimPathToSave=OTHER_FS_PATH\fR
```
Comment (veloman-yunkan) [inline src/man/kiwix-manage.1:51]: What about displaying `OTHER_FS_PATH` in regular or italic font?

File: src/man/kiwix-manage.1 [hunk 5]
```diff
@@ -1,62 +1,72 @@
...
-.br
-	kiwix-manage LIBRARY_PATH show [CONTENTID1] [CONTENTID2] ...
-.br
-	kiwix\-manage LIBRARY_PATH remove CONTENTID1 [CONTENTID2]
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH add ZIM_PATH ...
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH show [ZIM_ID_1] [ZIM_ID_2] ...
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH remove ZIM_ID_1 [ZIM_ID_2] ...
```
Comment (veloman-yunkan) [inline src/man/kiwix-manage.1:13]: What about using bold font for the actions (add, show, remove)?

**Review by veloman-yunkan — CHANGES_REQUESTED** [review]

File: src/man/kiwix-manage.1 [hunk 6]
```diff
@@ -1,62 +1,72 @@
...
 .TP
-\fBurl\fR
-Set the content location of the ZIM file over the network for in\-kiwix download.
+\fB\-\-version | \-v
+Display the version of kiwix-manage and all dependences
+
+.SH EXAMPLE
+Example library file can be found at https://download.kiwix.org/library/library_zim\.xml
+
+.SH SEE ALOS
```
[the remark this answers is not in the document]
Comment (kelson42) [reply src/man/kiwix-manage.1:68]: yes, fixed

File: src/man/kiwix-manage.1 [hunk 7]
```diff
@@ -1,62 +1,72 @@
...
+Display the kiwix-manage help
 
 .TP
-\fBurl\fR
-Set the content location of the ZIM file over the network for in\-kiwix download.
+\fB\-\-version | \-v
+Display the version of kiwix-manage and all dependences
+
+.SH EXAMPLE
+Example library file can be found at https://download.kiwix.org/library/library_zim\.xml
```
[the remark this answers is not in the document]
Comment (kelson42) [reply src/man/kiwix-manage.1:66]: OK, let's delete it, not 100% sure myself.

File: src/man/kiwix-manage.1 [hunk 8]
```diff
@@ -1,62 +1,72 @@
...
+library. The library file is a flat XML file listing ZIM files with
+all necessary information like id, favicon, date, creator,
+description, filepath, title, url, etc.
+
+.SH ACTIONS
+
 .TP
 \fBadd\fR
-Add \fBZIM_FILE\fP to \fBLIBRARY_FILE\fP.
+Add \fBZIM_FILE\fP to \fBLIBRARY_FILE\f.
```
[the remark this answers is not in the document]
Comment (kelson42) [reply src/man/kiwix-manage.1:32]: yes, it's an error

File: src/man/kiwix-manage.1 (hunk 1 above)
Comment (kelson42) [reply src/man/kiwix-manage.1:58]: fixed

File: src/man/kiwix-manage.1 (hunk 2 above)
Comment (kelson42) [reply src/man/kiwix-manage.1:62]: fixed

File: src/man/kiwix-manage.1 (hunk 5 above)
Comment (kelson42) [reply src/man/kiwix-manage.1:13]: yes, sounds a good idea

File: src/man/kiwix-manage.1 (hunk 4 above)
Comment (kelson42) [reply src/man/kiwix-manage.1:51]: Not really sure about that... if OK for you would prefer (at least for the moment) to keep it like this.

File: src/man/kiwix-manage.1 (hunk 3 above)
Comment (kelson42) [reply src/man/kiwix-manage.1:47]: Not really sure about that... if OK for you would prefer (at least for the moment) to keep it like this.

File: src/man/kiwix-manage.1 (hunk 8 above)
[the remark this answers is not in the document]
Comment (veloman-yunkan) [reply src/man/kiwix-manage.1]: Did you miss point 2. or you don't like that proposal?

File: src/man/kiwix-manage.1 (hunk 3 above)
Comment (veloman-yunkan) [reply src/man/kiwix-manage.1:47]: It's OK

File: src/man/kiwix-manage.1 (hunk 4 above)
Comment (veloman-yunkan) [reply src/man/kiwix-manage.1:51]: It's OK

**Review by veloman-yunkan — COMMENTED** [review]
Please address point 2. in [my earlier comment](https://github.com/kiwix/kiwix-tools/pull/576#discussion_r969935771) and squash all commits into one.

File: src/man/kiwix-manage.1 (hunk 8 above)
[the remark this answers is not in the document]
Comment (kelson42) [reply src/man/kiwix-manage.1]: Definitly, sorry I have forgotten that point.

**Review by veloman-yunkan — APPROVED** [review]

**Comment by veloman-yunkan** [comment]
I believe that this PR should be merged as one squashed commit.

**Pull request merged**